public/apps/account/account-nav-button.tsx

```diff
@@ -167,12 +168,31 @@ export function AccountNavButton(props: {
       />
     </div>
   );
-  return (
-    <EuiHeaderSectionItemButton id="user-icon-btn">
+
+  const isPlacedInLeftNav = props.coreStart.uiSettings.get('home:useNewHomePage');
```

Comment:
@AMoo-Miki can core expose this as a constant somewhere? Since I think a lot of the upcoming changes will rely on this hard coded string, wdyt?

Reply:
It would have been nice if they had done that a long time ago. This is switch that has been overused for various features. As a result, it doesn't have a clear owner to expose it as a constant.